### DISCUSSION TITLE
can't pair with CC2531

### DISCUSSION THREAD

**Discussion by jamkas** [Koenkk/zigbee2mqtt#5930] [Devices]
Hi community!

I've been using Zigbee with OpenHab but as I wasn't satisfied with the OH Zigbee binding I decided to try Zigbee2MQTT.
My test system is a Raspberry PI 3 with freshly installed Raspberry Pi OS. My zigbee dongle is a CC2531 because I already had one so I wanted to do tests with this and at a later stage buy a better one if i decide to migrate all my devices. I tried the firmwares "CC2531_DEFAULT_20190608" and "CC2531_SOURCE_ROUTING_20190619"

Installation went without any problems and I can start Zigbee2mqtt. The MQTT part works as well because I can listen to zigbee2mqtt/# and receive the published messages. So no problem there.

If I try to pair a device absolutely nothing happens. I tried a Xiaomi Aqara button and two different Philips hue bulbs by factory resetting them.

I read about people solving this issue by using another physical USB port but I tried all four without success.

I considered, that my running Openhab might pair them before Zigbee2mqtt has the chance to do so but the problem persists even if I remove the Zigbee dongle from my Openhab Raspberry pi.

Here is the log when I start Zigbee2mqtt:
[zigbee2mqtt_log.txt](https://github.com/Koenkk/zigbee2mqtt/files/5879477/zigbee2mqtt_log.txt)

Can you help me with my issue? Do I need to buy a new Zigbee stick (I'd prefer not to buy new hardware until I succeeded with testing)

Thank you very much for your help!

**Comment by DrTron** [comment]
Maybe try to start zigbee2mqtt manually as root instead of the service? Just for testing.
Also, the firmware DEFAULT is the correct one, maybe just try to reflash.
Besides that, the CC2531 are not really that reliable, I've had a dud with those before. But if it worked with the OH binding, that does not seem to be the case.

**Comment by jamkas** [comment]
Thanks for your reply!
Unfortunately starting manually as root didn't change anything. Curious thing is that the onboard led behaves according to the disable_led: true/false setting so there is some communication with the stick.

I'm starting to think it is a hardware issue. I took a closer look at the stick and noticed six soldering pads between the chip and the antenna which look like there should be something on them. Is it normal that those pads are not occupied or did indeed a relevant component detach and fall off from the pcb? It's not the stick I use for OH but it has been lying around for quite a while. So maybe I accidentally destroyed something there.
![cc2531](https://user-images.githubusercontent.com/59607360/106024099-b8326180-60c7-11eb-94fd-6dd9820e8184.jpg)

**Reply by DrTron** [reply]
Really can't confirm at the moment. Looking at images on the web, it seems that there's really a component missing.
But what prevents you from trying the stick that you use with OH? At least you know that works and it should have the same firmware on it.

But if it's any help, I use z2m with Openhab and it works fine. For some reason it crashes whenever I restart OH, but that might have something to do with my zwave stick.

**Comment by jamkas** [comment]
I've a Telegesis stick on my OH system and I think it's not supported by Zigbee2MQTT. Also I want to keep the live system and the testing apart from each other. 
I now ordered the slaesh’s CC2652RB stick (I'd have liked the Electrolama but it was out of stock) and hope it's going to solve my problems. Hopefully it arrives next week as I'm eager to test it.

**Reply by DrTron** [reply]
I have that one. Works great, but range is not that much better (compared to CC2531) as I'd hoped. But with additional CC2530 routers that's no issue.

**Reply by jamkas** [reply]
My place is not very large so I think it should work if i place it centrally. Otherwise I'll use some routers as you suggested.
Anyway, thank you very much for supporting me! I really appreciate it!

**Reply by DrTron** [reply]
Probably depends more on the wall material and construction than the size of the place. Reinforced concrete walls are not very good for RF range ;-)
